flytekit/clis/sdk_in_container/run.py

```diff
@@ -493,7 +493,7 @@ def _update_flyte_context(params: RunLevelParams) -> FlyteContext.Builder:
     if output_prefix and ctx.file_access.is_remote(output_prefix):
         with tempfile.TemporaryDirectory() as tmp_dir:
             archive_fname = pathlib.Path(os.path.join(tmp_dir, "script_mode.tar.gz"))
-            compress_scripts(params.computed_params.project_root, str(archive_fname), params.computed_params.module)
+            compress_scripts(params.computed_params.project_root, str(archive_fname), list(sys.modules.values()))
```

Comment:
@pingsutw I'm concerned with this change. At this point, is the main workflow imported by `flytekit`?

Reply:
I added a new `get_all_modules` function that returns all modules. It adds the `params.computed_params.module` if it is not already imported.

Reply:
sgtm, thank you!